package.json

```diff
@@ -25,8 +25,8 @@
     "node": ">= 0.6"
   },
   "dependencies": {
-    "nomnom": ">= 1.5.x",
-    "JSV": ">= 4.0.x"
+    "nomnom": "^1.5.x",
```

Comment:
1.5.x is a bit odd...

Reply:
Lol good question, I have no idea. If someone wants to try locally and report back that would be great